### ISSUE TITLE
Chorioectopia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11615]
**Preferred term label:**
Chorioectopia 

**Synonyms**

**Definition (free text, please give PubMed ID)**
displacement  of the choroid layer of the eye

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
dystroglycanopathies (e.g., https://omim.org/clinicalSynopsis/253280)
https://uts.nlm.nih.gov/uts/umls/concept/C3278505
**Your nano-attribution (ORCID)**
